django_freeradius/migrations/0018_add_default_profile.py

```diff
@@ -0,0 +1,45 @@
+import swapper
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.db import migrations
+
+def add_default_profiles(apps, schema_editor):
+    RadiusProfile = swapper.load_model('django_freeradius', 'RadiusProfile')
+    default_profile = RadiusProfile.objects.filter(default=True)
+    if not default_profile.exists():
+        limited_user = RadiusProfile(name="Limited User", default=True,
+                                     daily_session_limit=10800,
+                                     daily_bandwidth_limit=3000000000)
+        limited_user.save()
+        power_user = RadiusProfile(name="Power User",
+                                   daily_session_limit=86400,
```

Comment:
double this just in case we ever get a user that stays connected 24 hours straight and the NAS doesn't calculate the time perfectly and may log them out